src/components/shared/ListItem/ListItem.tsx

```diff
@@ -0,0 +1,124 @@
+import { nanoid } from 'nanoid';
+import React, { ReactNode } from 'react';
+import { FormattedDateParts } from 'react-intl';
+
+import Tag, { Appearance as TagAppearance } from '~shared/Tag';
+import { Message, UniversalMessageValues } from '~types';
+import { MotionState, MOTION_TAG_MAP } from '~utils/colonyMotions';
+import { getMainClasses } from '~utils/css';
+import { formatText } from '~utils/intl';
+
+import styles from './ListItem.css';
+
+const displayName = 'ListItem';
+
+export enum ListItemStatus {
+  NeedAction = 'NeedAction',
+  NeedAttention = 'NeedAttention',
+  Draft = 'Draft',
+  // Default status, does not do anything
+  Defused = 'Defused',
+}
+
+interface FormattedDateProps {
+  parts: {
+    value: string;
+  }[];
+}
+
+const FormattedDate = ({ parts }: FormattedDateProps) => (
+  <>
+    <span className={styles.day}>{parts[2].value}</span>
+    <span>{parts[0].value}</span>
+  </>
+);
+
+interface ListItemProps {
+  /** Avatar to be displayed */
+  avatar?: ReactNode;
+  /** The date the corresponding event was created */
+  createdAt?: string | number | Date;
+  /** To be displayed at the end of the list item, e.g. a countdown timer */
+  extra?: ReactNode;
+  /** Metadata to be displayed beneath the item title. */
+  meta?: ReactNode;
+  /** A click handler for the list item */
+  onClick?: (
+    e: React.MouseEvent<HTMLDivElement> | React.KeyboardEvent<HTMLDivElement>,
+  ) => void;
+  /** An ItemStatus. Styles the list item's 'before' psuedo-element. */
+  status?: ListItemStatus;
+  /** The tag to be displayed next to the title */
+  tag?: MotionState;
+  /** A title */
+  title: Message;
+  /** Values for the react-intl interpolation */
+  titleValues: UniversalMessageValues;
+}
+
+const ListItem = ({
+  avatar,
+  createdAt,
+  extra,
+  meta,
+  onClick,
+  status = ListItemStatus.Defused,
+  title,
+  titleValues,
+  tag,
+}: ListItemProps) => {
+  const tagStyles = tag && MOTION_TAG_MAP[tag];
+  return (
+    <li>
+      <div
+        className={getMainClasses({}, styles, {
+          [ListItemStatus[status]]: !!status,
+        })}
+        onClick={onClick}
+        onKeyDown={onClick}
+        role="button"
+        tabIndex={0}
```

Comment:
Linter asks for this since we're making a non-interactive element interactive. I would've merged this div with the `<li>` wrapper but linter doesn't like that either (again, to do with making non-interactive elements interactive)